packages/example-app-base/src/components/templates/PageImageSet.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { BaseComponent, IBaseProps, IClassNames, customizable } from 'office-ui-fabric-react/lib/Utilities';
```

Comment:
Just out of curiosity... is this an issue only with the Styling import?

Reply:
I don't think it is an issue only with the styling import, but we need to take a deeper look at each case before moving forward.